src/Parser.php

```diff
@@ -1,67 +1,78 @@
-<?php namespace Sturgeon\PHPArse;
+<?php
+
+declare(strict_types=1);
+
+namespace PhpVersions\PHParse;
 
 use DOMDocument;
-use DOMXpath;
+use DOMXPath;
 
 class Parser
 {
     protected $xpath;
 
+    /** @var string */
+    private $html;
```

Comment:
Possible unused variable?

Reply:
Possibly, I'll see about removing it. or assigning it in the static method.